.eslintignore

```diff
@@ -1,3 +1,4 @@
 !.eslintrc.js
 src/vendor/*
+src/loaders/basis_transcoder.worker.js
```

Comment:
Snuck this in here. This worker is a large, minified vendor file. It adds 40% more time to the eslint check, so it's best to ignore it.